docs/js/features/odata/use-generated-client.md

```diff
@@ -318,6 +348,10 @@ BusinessPartner.requestBuilder()
   .ignoreVersionIdentifier();
 ```
 
+#### Deep Update
+
+It is also possible to update an entity together with related entities in a single request, same as [deep create](#deep-create).
+
```

Comment:
That is not true ;) Deep update still needs to be implemented and will only be available for OData v4. Maybe, again, just add a comment for future reference when somebody documents v4?